src/Microsoft.Health.Fhir.TemplateManagement/Utilities/StreamUtility.cs

```diff
@@ -74,5 +74,16 @@ public static string CalculateDigestFromSha256(byte[] content)
             hashedValue += string.Join(string.Empty, hashedStrings);
             return hashedValue;
         }
+
+        public static string CalculateDigestFromSha256(Stream stream)
+        {
+            using SHA256 mySHA256 = SHA256.Create();
+            string hashedValue = "sha256:";
+            byte[] hashData = mySHA256.ComputeHash(stream);
+            string[] hashedStrings = hashData.Select(x => string.Format("{0,2:x2}", x)).ToArray();
+            hashedValue += string.Join(string.Empty, hashedStrings);
+            return hashedValue;
+        }
```

Comment:
It seems these two functions share most logics. Could we convert the input stream to byte[] and call the original one?

Reply:
Yeah, they are sharing same logic in get the hash value.  Converting to byte array will introduce extra cost than computing from a stream directly. I feel it's fine to keep two methods here.